<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlertAddItem</class>
 <widget class="QWidget" name="AlertAddItem">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>462</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AlertAddItem</string>
  </property>
  <widget class="QPushButton" name="alert_btn_ok">
   <property name="geometry">
    <rect>
     <x>169</x>
     <y>140</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>444</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>增加指令</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>419</width>
      <height>42</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>指令头：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_01"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>参数备注：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_02"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>参数备注：</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ed_03"/>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
